inorbit_republisher/README.md

```diff
@@ -0,0 +1,166 @@
+# InOrbit republisher for ROS 2
+
+This directory includes a republisher that allows mapping from arbitrary ROS2 values to ``InOrbit`` [custom data](https://www.inorbit.ai/faq#publish-custom-data) key/value pairs for application-specific observability.
+
+Currently only mapping from ROS2 topics is supported. The republisher could be extended to map actions, services and parameters.
+
+## Usage
+
+Create a YAML config file specifying the mappings you would like to use using this format:
+
+```yaml
+  republishers:
+  - topic: "/fruits_per_cubic_m"
+    qos: 5
+    msg_type: "fruit_msgs/msg/Citrus"
+    mappings:
+    - field: "num_oranges"
+      mapping_type: "single_field"
+      out:
+        topic: "/inorbit/custom_data/0"
```

Comment:
```suggestion
        topic: "/inorbit/custom_data"
```

In ROS 2 the "0" at the end doesn't work. Can you change this in all shown example mappings?